src/WebJobs.Script/Utility.cs

```diff
@@ -644,10 +644,6 @@ internal static void VerifyFunctionsMatchSpecifiedLanguage(IEnumerable<FunctionM
                 {
                     throw new HostInitializationException($"Found functions with more than one language. Select a language for your function app by specifying {RpcWorkerConstants.FunctionWorkerRuntimeSettingName} AppSetting");
                 }
-                else
```

Comment:
Why are you deleting this? Isn't this .NET case you're looking at fall into the bucket of "single language"?

Reply:
yes. that is needed. I added it back in the last iteration, but now it conditionally throws based on the `throwOnMismatch` parameter value. With the new changes, we do not want to throw in the inproc payload + DNI FWRuntime value.